coalib/processes/Processing.py

```diff
@@ -1,10 +1,11 @@
-from itertools import chain
 import multiprocessing
 import os
 import platform
 import queue
 import subprocess
+from itertools import chain
```

Comment:
unrelated change + it breaks alphabetical order (I think we don't distinct between normal `import ...` and `from ... import ...`, just the module name matters)